ios/app/app-info.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>MGLMapboxAccessToken</key>
```

Comment:
Revert this change, per https://github.com/mapbox/mapbox-gl-native/pull/2437#issuecomment-143886759.